rust/arrow/src/lib.rs

```diff
@@ -139,15 +139,15 @@ mod arch;
 pub mod array;
 pub mod bitmap;
 pub mod buffer;
-pub mod bytes;
+mod bytes;
 pub mod compute;
 pub mod csv;
 pub mod datatypes;
 pub mod error;
 pub mod ffi;
 pub mod ipc;
 pub mod json;
-pub mod memory;
+mod memory;
```

Comment:
@jorgecarleitao 

This change has broken a bunch of our code downstream as we build buffers directly. Can you make the memory module public again?